docs/devupdate.md

```diff
@@ -121,3 +121,11 @@ Please see the [PHPUnit 11 Upgrade Guide](/general/development/tools/phpunit/upg
 <Since version="5.0" issueNumber="MDL-79843" />
 
 A new process for plugin type and subplugin type deprecation has been introduced. See [the Plugin Type deprecation](./apis/plugintypes/index.md#deprecating-a-plugin-type) and [Subplugin deprecation](./apis/plugintypes/index.md#deprecating-a-subplugin-type) documentation for further information.
+
+## Activity overview page integration
```

Comment:
Thanks for adding this section to de devupdate.md page.
The sections on this page are ordered alphabetically so this should appear as the first one ;-) 

Could you please move it? Thanks :-)